doc/migration/version-upgrade-guide.md

```diff
@@ -168,8 +168,139 @@ The following sections outline the upgrade steps for each SDK version along with
+- Ensure that you can build the project with `operator-sdk build`
+- Finally remove `Gopkg.lock`, `Gopkg.toml` and the vendor directory.
+
+## `v0.9.x`
+
+- The function `ExposeMetricsPort()` has been replaced with `CreateMetricsService()` [#1560](https://github.com/operator-framework/operator-sdk/pull/1560). 
+  
+  Replace the following line in `cmd/manager/main.go`
+  ```Go
+    _, err = metrics.ExposeMetricsPort(ctx, metricsPort)
+  ```
+  with
+  ```Go
+	servicePorts := []v1.ServicePort{
+    {Port: metricsPort, Name: metrics.OperatorPortName, Protocol: v1.ProtocolTCP, TargetPort: intstr.IntOrString{Type: intstr.Int, IntVal: metricsPort}},
+    {Port: operatorMetricsPort, Name: metrics.CRPortName, Protocol: v1.ProtocolTCP, TargetPort: intstr.IntOrString{Type: intstr.Int, IntVal: operatorMetricsPort}},
+  }
+
+  _, err = metrics.CreateMetricsService(ctx, servicePorts)
+  ```
+
+### `dep`
+
+- Update the SDK constraint in `Gopkg.toml` to version `v0.9.0`, the kubernetes dependencies to `kubernetes-1.13.4` revisions, and the controller-runtime version to `v0.1.12`.
+  ```TOML
+  [[override]]
+    name = "k8s.io/api"
+    # revision for tag "kubernetes-1.13.4"
+    revision = "5cb15d34447165a97c76ed5a60e4e99c8a01ecfe"
+  [[override]]
+    name = "k8s.io/apiextensions-apiserver"
+    # revision for tag "kubernetes-1.13.4"
+    revision = "d002e88f6236312f0289d9d1deab106751718ff0"
+  [[override]]
+    name = "k8s.io/apimachinery"
+    # revision for tag "kubernetes-1.13.4"
+    revision = "86fb29eff6288413d76bd8506874fddd9fccdff0"
+  [[override]]
+    name = "k8s.io/client-go"
+    # revision for tag "kubernetes-1.13.4"
+    revision = "b40b2a5939e43f7ffe0028ad67586b7ce50bb675"
+  [[override]]
+    name = "github.com/coreos/prometheus-operator"
+    version = "=v0.29.0"
+  [[override]]
+    name = "sigs.k8s.io/controller-runtime"
+    version = "=v0.1.12"
+  [[constraint]]
+    name = "github.com/operator-framework/operator-sdk"
+    version = "=v0.9.0"
+  ```
+- Append the contraint for `k8s.io/kube-state-metrics`.
+  ```TOML
+  [[override]]
+    name = "k8s.io/kube-state-metrics"
+    version = "v1.6.0"
+  ```
+- Run `dep ensure` to update the vendor directory.
+
+### modules
+
+- Update the `replace` directives in your `go.mod` file for the SDK, kubernetes, controller-runtime and kube-state metrics dependencies to the following versions.
+  ```
+  // Pinned to kubernetes-1.13.4
+  replace (
+    k8s.io/api => k8s.io/api v0.0.0-20190222213804-5cb15d344471
+    k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190228180357-d002e88f6236
+    k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
+    k8s.io/client-go => k8s.io/client-go v0.0.0-20190228174230-b40b2a5939e4
+  )
+  replace (
```

Comment:
I think it need be updated as here; https://github.com/operator-framework/operator-sdk/pull/1899

Reply:
My first thought is probably not.
There are other ways to solve this problem, e.g using the proxy cache by setting `GOPROXY=https://proxy.golang.org`.

I don't want to add it to the guide since this isn't an issue specifically with the SDK and something that's easily if you run into the issue.

Although it looks like https://status.apache.org/ is down quite often so maybe if we merge #1899 I can do a follow up.